airbyte-integrations/connectors/source-relational-db/src/main/java/io/airbyte/integrations/source/relationaldb/InvalidCursorInfoUtil.java

```diff
@@ -15,14 +15,15 @@ public static String getInvalidCursorConfigMessage(final List<InvalidCursorInfo>
             .collect(Collectors.joining(","));
```

Comment:
Do we want to update the messaging here? Probably want to seperate out tables that have an invalid cursor altogether vs NULL values so that the user can quickly get back into a reasonable state

Reply:
I did that - now giving a cause for failure. Either "Unsupported cursor type" or "Cursor column contains NULL value" which is formatted to the error shown in UI.
This the only place this code is used so I didn't bother with creating subclasses structure.
Added a new cause field instead

Reply:
My suggestion would be to change 
"The following tables have invalid columns selected as cursor, please select a column with a well-defined ordering as a cursor" -> "The following tables have invalid columns selected as cursor or a NULL value in the cursor field. Please select a column with a well-defined ordering as a cursor and make sure it doesn't contain NULL values: "

WDYT? Bit verbose and the message you added to the individual `InvalidCursorInfo` would help pinpoint the exact table

Reply:
> invalid columns selected as cursor **or** a NULL value in the cursor field
I tried to specify what the exact error is so the user will be able to know the exact reason we failed.
See at the end of the [loom](https://www.loom.com/share/f410403f19324a40b0d0f51c80d2f87c) I recorded. 
Maybe I'm not getting it, but is your suggestion improve on current behavior?

Reply:
Maybe just add this line : "The following tables have invalid columns selected as cursor, please select a column with a well-defined ordering as a cursor and **a non-null value**"

Just a quick update to the existing string in line 13. But I'll leave it to you

Reply:
Added